doc/stdenv/meta.xml

```diff
@@ -190,7 +190,7 @@ hello-2.3  A program that produces a familiar, friendly greeting
     <listitem>
      <para>
       A list of names and e-mail addresses of the maintainers of this Nix expression. If you would like to be a maintainer of a package, you may want to add yourself to <link
```

Comment:
```suggestion
      A list of the maintainers of this Nix expression. Maintainers are defined in <link xlink:href="https://github.com/NixOS/nixpkgs/blob/master/maintainers/maintainer-list.nix"><filename>nixpkgs/maintainers/maintainer-list.nix</filename></link>. Add each new maintainer to that list in a separate commit titled 'maintainers: add alice', and reference maintainers with <literal>maintainers = with lib.maintainers; [ alice bob ]</literal>.
```

The previous wording was very wishy-washy. I prefer the instructions being very concrete.

Reply:
While I agree this sentence can probably be improved further, I'm reluctant to delay this PR even more by increasing its scope.

It is probably good to keep the information that it is OK to add yourself as a maintainer - that seems lost in your proposal.

Reply:
I thought it was implied by "each new maintainer". However I guess we can be explicit that there is no barrier to becoming a new maintainer.

Ok, even if you don't want to update the previous sentence in this PR I still would like to clean up the modified sentence to be a bit more direct.